openapi.yaml

```diff
@@ -1016,18 +1016,39 @@ components:
           properties:
             code:
               enum: [NOSU]
+    JobSuffixRequired:
+      type: object
+      required:
+        - suffix
+      properties:
+        suffix:
+          type: string
+          nullable: false
+          maxLength: 2
+          description: >-
+            Used to uniquely identify a job record for persons who have multiple
+            jobs that share the same position number. This field is required.
+    JobSuffixOptional:
+      type: object
+      properties:
+        suffix:
+          type: string
+          nullable: true
+          maxLength: 2
+          description: >-
+            Used to uniquely identify a job record for persons who have multiple
+            jobs that share the same position number. This field is optional.
```

Comment:
Why do you need to create two exact same attributes and the only difference is `nullable: true`? Can you think of a better way to improve it?